tests/tensors/backend_test.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=no-self-use,invalid-name
+import numpy
+from numpy.testing import assert_allclose
+from keras import backend as K
+
+from deep_qa.tensors.backend import cumulative_sum
+from ..common.test_case import DeepQaTestCase
+
+
+class TestBackendTensorFunctions(DeepQaTestCase):
+    def test_cumulative_sum(self):
```

Comment:
sorry, i know it's annoying but can you add a quick test for the masked case as well?